dev-mode/.envrc

```diff
@@ -0,0 +1 @@
+AWS_PROFILE=sls-test
```

Comment:
This looks environment specific, do we need to commit it in?

Reply:
Huh I have this in the gitignore so I am not sure how this got added 🤔 Anyway you are correct this should be removed

Reply:
This should be gone now 😎